openfisca_country_template/entities.py

```diff
@@ -29,7 +44,20 @@
     key = "person",
     plural = "persons",
     label = u'Person',
+    doc = textwrap.dedent('''
+    A Person represents a natural person, the minimal legal entity on which a legislation might be applied.
+
+    Example:
+    'salary' and 'income_tax' variables are usually defined for the entity 'Person'.
```

Comment:
The 'salary' and 'income_tax' variables are usually defined for the entity 'Person'.